v03_pipeline/lib/reference_data/config.py

```diff
@@ -102,10 +102,11 @@ def custom_gnomad_select_v2(ht):
     """
     selects = {}
     global_idx = hl.eval(ht.globals.freq_index_dict['gnomad'])
-    selects['AF'] = hl.float32(ht.freq[global_idx].AF)
-    selects['AN'] = ht.freq[global_idx].AN
-    selects['AC'] = ht.freq[global_idx].AC
-    selects['Hom'] = ht.freq[global_idx].homozygote_count
+    freq = ht.freq[global_idx]
```

Comment:
can we actually run a test to make sure  that this is faster?  We can do it locally with a hl.Table.parallelize and just see what hail logs.

Reply:
mind showing me how you'd do this?

Reply:
Yeah maybe something like:

```
table1 = hl.Table.parallelize(...)
table2 = hl.Table.parallelize(...)
selects = {}
select['a'] = table1[table2.key].a
select['a'] = table1[table2.key].a

Reply:
thanks, how would you run it/see the logs?